src/components/HyloEditor/HyloContentState.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import { trim } from 'lodash/fp'
+import Immutable from 'immutable'
+import { convertToHTML, convertFromHTML } from 'draft-convert'
+import { convertToRaw, convertFromRaw } from 'draft-js'
+import {
+  PathHelpers,
+  MENTION_ENTITY_TYPE,
+  TOPIC_ENTITY_TYPE,
+  ALL_GROUPS_CONTEXT_SLUG
+} from 'hylo-shared'
+
+// NOTE: Legacy mention links are in this format --
```

Comment:
should we do a database migration to update/fix?

Reply:
To this and the below. No, certainly not with this update. There is already a lot of change to let settle. These things aren't having much impact and won't cause a different result on display right now. Maybe another pass another time if it comes-up and at that time we may have other content transformations that come-up to apply then?